target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -58,6 +58,11 @@ comfast,cf-e5)
 dlink,dir-859-a1)
 	ucidef_set_led_switch "internet" "WAN" "$boardname:green:internet" "switch0" "0x20"
 	;;
+engenius,ecb1750)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:blue:lan" "eth0"
+	ucidef_set_led_wlan "wlan2g" "WLAN 2.4 GHz" "$boardname:blue:wlan2g" "phy1tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN 5 GHz" "$boardname:blue:wlan5g" "phy0tpt"
+	;;
 engenius,ews511ap)
```

Comment:
Are the definitions for wireless leds needed if You are already specifying the proper trigger in dts?

Reply:
You are right.. they are not needed.